src/DynamoCoreWpf/Extensions/ViewExtensionDefinition.cs

```diff
@@ -10,9 +11,9 @@ namespace Dynamo.Wpf.Extensions
     internal class ViewExtensionDefinition
     {
         [DataMember]
-        public string AssemblyName { get; set; }
+        public string AssemblyPath { get; set; }
 
         [DataMember]
-        public string TypeName { get; set; }
+        public string ExtensionDefinition { get; set; }
```

Comment:
Both `ViewExtension` definition and `Definition` should have precisely the same names!